.gitignore

```diff
@@ -94,3 +94,7 @@ prime/
 
 # Make evidence files
 /.make_evidence
+
+# Manpage
+man/man1/gitea.1
```

Comment:
why are this files still in this pull diff?

Reply:
@6543 i think this PR build manpages under `man/` so I had to `.gitignore` resulting files.